requirements.txt

```diff
@@ -1,9 +1,9 @@
-coincurve>=8.0.0,<9.0
+coincurve>=9.0.0,<10.0
 eth-utils>=1.2.0,<2.0
 gevent>=1.3.6,<2.0
 jsonschema>=2.6.0,<3.0
 matrix_client>=0.3.2,<4.0
 pycryptodome>=3.6.6,<4.0
-web3>=4.5.0,<5.0
+web3>=4.7.0,<5.0
```

Comment:
why not 4.7.1? https://github.com/raiden-network/raiden/blob/b1867299a8de793a4991ae560abc9898c6c6673b/constraints.txt#L80